lib/archethic/db/embedded_impl/chain_index.ex

```diff
@@ -226,10 +220,14 @@ defmodule Archethic.DB.EmbeddedImpl.ChainIndex do
   def get_tx_entry(address, db_path) do
     case LRU.get(@archetic_db_tx_index_cache, address) do
       nil ->
-        # If the transaction is not found in the in memory lookup
-        # we scan the index file for the subset of the transaction to find the relative information
-        # This will update the LRU cache if the transaction is found
-        search_tx_entry(address, db_path)
+        case search_tx_entry(address, db_path) do
+          {:ok, entry} ->
+            LRU.put(@archetic_db_tx_index_cache, address, entry)
```

Comment:
This should be also conditionned by `Archethic.up?` I think, as when a node do its bootstrap_sync, the function `get_tx_entry` is often called